benchexec/tools/fusebmc.py

```diff
@@ -45,10 +51,4 @@ def determine_result(self, run):
         if run.output.any_line_contains("DONE"):
             status = result.RESULT_DONE
 
-        if status == result.RESULT_UNKNOWN:
-            if run.was_timeout:
-                status = "TIMEOUT"
-            elif not run.output.any_line_contains("Unknown"):
```

Comment:
It would be good if the tool-info module would still be able to recognize the difference between "unknown" (tool gave up because it could not compute an answer) and "error" (unexpected problem occurred, bug in the tool, crash, etc.). Only the check for timeout was redundant.

Of course, if fusebmc never outputs `Unknown` then the current logic is fine.

Reply:
I would say fusebmc never outputs Unknown.